packages/javascript/bh-shared-ui/src/utils/finishedJobs.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright 2025 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import type { GetScheduledJobDisplayResponse, ScheduledJobDisplay } from 'js-client-library';
+import { DateTime, Interval } from 'luxon';
+import type { OptionsObject } from 'notistack';
+import { useEffect } from 'react';
+import { useQuery } from 'react-query';
+import type { StatusType } from '../components';
+import { usePermissions } from '../hooks';
```

Comment:
For some reason, using this import from `hooks` causes `TagForm.test.tsx` to fail. So this was changed to include the module.

```suggestion
import { usePermissions } from '../hooks/usePermissions';
```